keyboards/sporewoh/banime40/readme.md

```diff
@@ -0,0 +1,27 @@
+# banime40 
+
+<img src="https://raw.githubusercontent.com/ChrisChrisLoLo/banime40/master/docs/images/IMG_20220319_144006.jpg" width="500">
```

Comment:
Should be reuploaded to imgur.

Reply:
Good catch, I've now uploaded the image imgur